### ISSUE TITLE
Health terms and definitions

### ISSUE THREAD

**Issue by lemaka** [cwrc/ontology#285]
Draft: 
- [ ] Orlando Sub-Element
- [ ] Class
- [ ] Predicate
- [ ] Inverse Predicate
- [ ] Attributes
- [ ] Attribute Values
- [ ] Instances
- [ ] Label
- [ ] Ontology relationships
- [ ] Definitions from Orlando
- [ ] Definition for Ontology
- [ ] French Label
- [ ] French Translation
- [ ] Examples from Orlando
- [ ] Instance in the UPTODATE data file
- [ ] Extraction Location
- [ ] Xpath indication 

https://docs.google.com/spreadsheets/d/1L9V__vfiZ2hEZOsbVkMZ7-JlsToHHOuOKzbPIi-HTPs/edit?usp=sharing

**Comment by lemaka** [comment]
@SusanBrown labels are ready for review

**Comment by lemaka** [comment]
**How we are going about the data in Health:**

Deb: Health is going to be hard to parse. There are 3 things that go into Health issues (female body, mental, and physical health). Use Cause of Death tags to do the same kind of work? But it would be spotty. A lot of manual work would be needed. 

**Susan (Decision): We limit it to Events, bc then it has a chronprose and would only have a few sentences -- would drastically reduce the amount of data and improve the quality.** 

Perhaps: HealthContext; BodilyHealthConcern; hasFactor -- Cancer

use 160 terms in the WHO vocabulary -- we will import this taxonomy and cite them

**Comment by alliyya** [comment]
Currently not extracting health factors, no other definitions currently needed.
